modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/indices.analyze/10_analyze.yml

```diff
@@ -19,10 +19,10 @@
     - skip:
         version: " - 6.2.99"
         reason: deprecated in 6.3
-        features: "warnings"
+        features: "allowed_warnings"
```

Comment:
I admit that I don't really understand what these files mean, but why did you change "warnings" to "allowed_warnings"?

Reply:
`allowed_warnings` asserts if the warning occurs but does not fail the test if it doesn't. This is in a lot of files and there is no order in which they run so it is very difficult to determine when the first warning will occur given that a lot of tests  assert the same duplicate warnings.